.travis.yml

```diff
@@ -9,19 +9,18 @@ dart:
 dart_task:
   - test: --platform vm
 
+branches:
+  only:
+  - master
+
```

Comment:
This is unlikely to build, and frankly I'd rather not have pushes to master at all. I'm going to try to wrap up the discussion about protecting master via repo settings and automate this change across all the tracks.